From 03bb6b20a8eaa7c565401597294ea9e3aee6d4ab Mon Sep 17 00:00:00 2001
From: Andrea Ambu <andreaambu@google.com>
Date: Thu, 8 Dec 2022 17:22:37 +0000
Subject: [PATCH] speech: No BIND_ALLOW_BACKGROUND_ACTIVITY_STARTS

Bug: 232799171
Test: cts
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:433ba102f30f0ab07e8ac0ff721ce97610eff030)
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:02ce77179cdc824208deb78c79202cbee6d98a36)
Merged-In: I72114a936682d735bbe4f79e2c43bba2cdade97e
Change-Id: I72114a936682d735bbe4f79e2c43bba2cdade97e
---
 .../android/server/speech/RemoteSpeechRecognitionService.java  | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/services/core/java/com/android/server/speech/RemoteSpeechRecognitionService.java b/services/core/java/com/android/server/speech/RemoteSpeechRecognitionService.java
index f4b335e42ec5..31e5c1ba7673 100644
--- a/services/core/java/com/android/server/speech/RemoteSpeechRecognitionService.java
+++ b/services/core/java/com/android/server/speech/RemoteSpeechRecognitionService.java
@@ -69,8 +69,7 @@ final class RemoteSpeechRecognitionService extends ServiceConnector.Impl<IRecogn
                 new Intent(RecognitionService.SERVICE_INTERFACE).setComponent(serviceName),
                 Context.BIND_AUTO_CREATE
                         | Context.BIND_FOREGROUND_SERVICE
-                        | Context.BIND_INCLUDE_CAPABILITIES
-                        | Context.BIND_ALLOW_BACKGROUND_ACTIVITY_STARTS,
+                        | Context.BIND_INCLUDE_CAPABILITIES,
                 userId,
                 IRecognitionService.Stub::asInterface);
 
-- 
2.49.0.395.g12beb8f557-goog

